src/main/java/net/snowflake/client/jdbc/SnowflakeFileTransferMetadataV1.java

```diff
@@ -50,7 +51,10 @@ public SnowflakeFileTransferMetadataV1(
 
   @Override
   public boolean isForOneFile() {
```

Comment:
Is this method only use by Spark connector?

Reply:
yes.